mesos-simulation/src/main/scala/mesosphere/mesos/simulation/DriverActor.scala

```diff
@@ -2,14 +2,14 @@ package mesosphere.mesos.simulation
 
 import java.util.UUID
 
-import akka.actor.{ Actor, ActorRef, Cancellable, Props }
+import akka.actor.{Actor, ActorRef, Cancellable, Props}
```

Comment:
Recompiling is warranted :)

Reply:
Do you know which part of linter/formatter does that? I remember trying to disable this once but I failed :(